### DISCUSSION TITLE
edit "send later" messages after they are created - how?

### DISCUSSION THREAD

**Discussion by Stefanseiner** [Extended-Thunder/send-later#540] [Q&A] [answered]
Hello,
is there any possibility to change / edit messages once they are prepared to "send later" ?
I would like to change text and time.
Thanks and regards
Stefan

**Comment by jikamens** [comment, marked as answer]
Scheduled messages are in your Drafts folder. You can edit them there like any other Draft. When you open a draft for editing, the scheduled send is automatically cancelled, so if you want it to be a scheduled message after editing it, you need to schedule it again.
